hubploy/auth.py

```diff
@@ -69,49 +70,78 @@ def registry_auth_gcloud(deployment, project, service_key):
     yield
 
 
-def registry_auth_aws(deployment, project, zone, service_key):
+def registry_auth_aws(deployment, project, zone, service_key=None, role=None):
```

Comment:
can we call this role_arn?

Reply:
@yuvipanda had requested "role", but it doesn't matter to me.

Reply:
role_arn is more specific, let's use that?